mne_bids/write.py

```diff
@@ -187,12 +190,60 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
                             ('z', z),
                             ('size', sizes),
                             ])
-    else:
+    elif kind == 'eeg':
+        picks = pick_types(raw.info, eeg=True)
+        ch_locs = []
+        for ind, (name, _x, _y, _z) in enumerate(zip(names, x, y, z)):
+            if ind not in picks:
+                ch_locs.append([np.nan, np.nan, np.nan])
+            elif "n/a" in [_x, _y, _z]:
+                warn("{} electrode has n/a coordinate. "
```

Comment:
1. did you fortget a `.format` for this warning?
1. can we use the logger to emit a warning?